components/invitation_modal/__snapshots__/invitation_modal_members_step.test.jsx.snap

```diff
@@ -57,7 +62,7 @@ exports[`components/invitation_modal/InvitationModalMembersStep should match the
       className="help-text"
     >
       <FormattedMessage
-        defaultMessage="Share this link to grant member access to this team."
+        defaultMessage="Share this link to grant access to this team."
```

Comment:
It's not exactly "granting access", since they still have to create an account. Perhaps "invite people" is more appropriate?

Reply:
+1: Reviewed with @michaelgamble  - Please update this text to  "Share this link to invite people to this team"